functions/src/environment.ts

```diff
@@ -0,0 +1,10 @@
+import * as functions from "firebase-functions";
+
+let config = process.env;
+// use firebase config when deployed to firebase
+const deployedToFirebase = process.env.NODE_ENV === "production";
+if (deployedToFirebase) {
+  config = functions.config().env;
+}
+
+export { config as environment };
```

Comment:
@harshasrikara This is the environment file.

Reply:
Missed seeing this in my first review, oops 😅